src/app/[locale]/(private)/module/msg/components/dialog/preview-dialog.tsx

```
@@ -0,0 +1,55 @@
+                    {formatDate(selectedMail.createdAt)} {formatTime(selectedMail.createdAt)}
+                  </Paragraph>
+                </div>
+              </div>
+              <div className="text-muted-foreground text-sm">
+                <Paragraph>
+                  <span className="font-medium">{t('dialog.to')}</span> {selectedMail.recipient.name}
+                </Paragraph>
+              </div>
+              <div
+                className="pt-2 text-base leading-relaxed"
+                dangerouslySetInnerHTML={{ __html: selectedMail.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)